### ISSUE TITLE
Description of variant ids

### ISSUE THREAD

**Issue by oleraj** [jandrewrfarrell/RUFUS#12]
Hi,

Can you explain what is the difference between the different variant ids?  I don't see any documentation about this.  For example, here are the most common variant types I found in one particular trio:
```
    187 X-DeNovo
     47 X-Mosaic
     16 I-DeNovo
     10 D-DeNovo
      4 3D-Mosaic
      4 3I-Mosaic
      4 DD-DeNovo
      3 4D-DeNovo
      3 DD-Mosaic
      3 I-Mosaic
      3 XX-Mosaic
      2 12D-Mosaic
      2 13D-DeNovo
      2 3D-DeNovo
      2 4I-DeNovo
      2 5D-DeNovo
      2 6I-DeNovo
      2 D-Mosaic
      1 10I-DeNovo
      1 15I-DeNovo
```

I think I can infer that X-DeNovo and X-Mosaic are single nucleotide de novo/mosaic variants while XX-DeNovo and XX-Mosaic are 2 nucleotide variants MNPs; I-DeNovo and D-DeNovo are single base insertion and deletion; 4D-DeNovo is a 4nt deletion; DD-DeNovo is a 2nt deletion (why not 2D-Denovo?); etc. -- is that correct?  Might be good to put that in the documentation somewhere.  I was able to figure these out because I had enough examples from the whole genome run to figure out the pattern but when I first ran it with just an exome I only got 2 de novo variants so I was really confused.

Thanks,

Andrew

**Comment by jandrewrfarrell** [comment]
yup, that is correct.  Good point, i'll add something about that to the
documentation
